Dockerfile

```diff
@@ -1,28 +1,21 @@
-FROM elifesciences/python:b22fbad8f66100d88cb4bd7c7d092b376a9e09bc
+ARG image_tag=latest
+FROM elifesciences/profiles_venv:${image_tag} as venv
+ARG python_version
+FROM elifesciences/python:${python_version}
```

Comment:
Will be changed when https://github.com/elifesciences/elife-base-images/pull/20 is merged.